docs/src/main/asciidoc/ghpages.sh

```diff
@@ -134,44 +135,17 @@ function add_docs_from_target() {
 
 # Copies the docs by using the retrieved properties from Maven build
 function copy_docs_for_current_version() {
-    if [[ "${CURRENT_BRANCH}" == "main" ]] ; then
-        echo -e "Current branch is main - will copy the current docs only to the root folder"
+    if [[ "${RELEASE_TO_GH_PAGES_ENABLED_VALUE}" == "true" ]] ; then
+        echo -e "Need to release - will copy the current docs only to the root folder"
```

Comment:
`will copy the current docs only to the root folder` -> `Will copy the all files from "docs/target/generated-docs/ to ${ROOT_FOLDER}${RELEASE_TO_GH_PAGES_TARGET_FLODER_VALUE} "`